include/RAJA/config.hpp.in

```diff
@@ -263,6 +263,7 @@ namespace RAJA {
 // enable device function pointers with rocm version >= 5.1
 // this used to be set to 4.3, but tests start passing with 5.1
 #define RAJA_ENABLE_HIP_INDIRECT_FUNCTION_CALL
+#define RAJA_ENABLE_HIP_DOUBLE_ATOMICADD
```

Comment:
Can we make this the default when on the supported platforms?

Reply:
This is based on rocm version, not device/platform.

Reply:
Thanks I see it now, 5.1 and above right? Nice feature, I would have thought there might be a dependency on the compute arch number.